libs/solid/skeleton/src/lib/services/feedback.service.ts

```diff
@@ -35,19 +35,17 @@ export class FeedbackService {
       location: location,
       title: location ? 'Fehler melden' : 'Kontakt und Feedback',
       subject: location ? 'Fehler melden' : 'Feedback',
-      message: location ? '' + '\n\nFehler gefunden in: ' + location : ''
+      message: location ? '' + '\n\nFehler gefunden in: ' + location : '',
     };
     dialogConfig.width = '80%';
     dialogConfig.maxWidth = '600px';
     this._dialog.open(FeedbackComponent, dialogConfig);
   }
 
   public submitFeedback(value: any) {
-    this._http
-      .post<unknown>(`${this._config.apiUrl}/contact`, value)
-      .pipe(
-        map((_) => true),
-        catchError((err) => of(false))
-      );
+    this._http.post<unknown>(`${this._config.apiUrl}/contact`, value).pipe(
```

Comment:
Hm... sieht alles gut aus, aber es passiert nichts. 
Wenn Du im Network-Tab (DevTools) nachschaust, müsste eigentlich der Api-Call auf contact sichtbar sein. Aber er taucht nicht auf.

Reply:
Ich habe auch nochmal probiert und `_http.post...` wird nicht ausgeführt. 
Mit Observable funktioniert es sofort 🤷

Reply:
Ich habe das Konzept beim Observable und HTTP nicht so gut verstanden, deswegen habe ich ein Fehler gemacht... also nun wenn ich den Code modifiziert, kann ich der API-Call sehen, bekomme ich aber diese Meldung:
![Screenshot (51)](https://user-images.githubusercontent.com/33594579/157845005-b0d9c94e-e9e1-4abd-a483-19e0373a6670.png)

Reply:
Ich glaube, der Fehler ist "normal" - den bekomme ich auch in `dev ` mit dem alten Code.
Ich verstehe es auch nicht ganz genau, denn dieses GET führen wir eigentlich gar nicht aus, aber es kann daran liegen, dass wir lokal arbeiten.

Reply:
Wenn es auf dev genauso aussieht, würde ich erwarten, dass es okay ist. Im Network-Tab siehst Du mehr Details:

![image](https://user-images.githubusercontent.com/13869236/157847380-5ddbb24c-a43c-463d-a96e-ed5980299e31.png)